src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -106,6 +129,11 @@ public void WriteInformation(string message)
             }
         }
 
+        /// <summary>
+        /// Writes an information event log without generating an event.
```

Comment:
should be same "summary" as a above. Only difference is 'formatting parameters' are passed.